### PR TITLE
[BREAKING CHANGE] Support browserlist configuration

### PR DESCRIPTION
**Pull request by moonglum**
As suggested by @FND in #29 we could go the standard route of configuring the browsers you want to target. With this PR, instead of defining a `prefix` option in your config, faucet-pipeline-sass will autodetect browserslists. If it finds one, it will activate autoprefixing – if it doesn't find one, it will not autoprefix at all. You can provide the browserslists in the ways defined by [browserslist](https://github.com/ai/browserslist/#packagejson):

* `.browserslistrc`
* `browserslist`
* `package.json` with `browserslist`

It also raises an exception if you use the `prefix` option so we can get rid off it before 1.0.0

### CODE REVIEW

File: lib/build-bundler.js [hunk 1]
```diff
@@ -69,8 +71,14 @@ function buildSassRenderer({ inputFileName, outputStyle, includePaths }) {
...
-function buildAutoprefixer({ prefixes }) {
-	let processor = postcss([ autoprefixer(prefixes) ]);
+function buildAutoprefixer({ configDir }) {
+	let browsers = browserslist.findConfig(configDir);
+	let plugins = [];
+	if(browsers) {
+		plugins.push(autoprefixer({ browsers: browsers["defaults"] }));
+	}
+
+	let processor = postcss(plugins);
```
Comment (FND) [inline lib/build-bundler.js]: If `browsers` is falsy, do we need to invoke PostCSS at all?

Comment (moonglum) [reply lib/build-bundler.js]: Agreed. But I'm still thinking about offering [other PostCSS plugins](https://github.com/postcss/postcss). Therefore I'll keep it like this for now.

Comment (FND) [reply lib/build-bundler.js]: YAGNI?
WTFAYD

...

fine, ¯\\\_(ツ)_/¯

Comment (moonglum) [reply lib/build-bundler.js]: ❤️

**Pull request merged**